lib/sqlalchemy/orm/session.py

```diff
@@ -3204,6 +3207,14 @@ def __repr__(self):
         )
 
 
+def close_all():
+    """Close *all* sessions in memory.
```

Comment:
**mike bayer** (zzzeek) wrote:

"""Close all sessions in memory.

This function consults a global registry of all :class:`.Session` objects and calls :meth:`.Session.close`
on them, which resets them to a clean state.

This function is not for general use but may be useful
for test suites within the teardown scheme.

.. versionadded:: 1.3
"""